quality/detekt.yml

```diff
@@ -175,35 +176,16 @@ complexity:
   LongMethod:
     active: true
     threshold: 20
-  NestedBlockDepth:
-    active: true
-    threshold: 3
-  LongParameterList:
-    active: true
-    threshold: 5
   LargeClass:
     active: true
     threshold: 150
   ComplexInterface:
     active: false
     threshold: 10
     includeStaticDeclarations: false
-  ComplexMethod:
-    active: true
```

Comment:
nit: active: false instead, here and below? I think we should still keep all the checks in the config, just so it's easier to turn them on again